browser/brave_wallet/brave_wallet_service_unittest.cc

```diff
@@ -1172,6 +1256,69 @@ TEST_F(BraveWalletServiceUnitTest, ERC721TokenAddRemoveSetUserAssetVisible) {
   EXPECT_EQ(erc721_token_1_visible_false, tokens[1]);
 }
 
+TEST_F(BraveWalletServiceUnitTest, SolanaTokenUserAssetsAPI) {
+  bool success = false;
+  std::vector<mojom::BlockchainTokenPtr> tokens;
+
+  GetUserAssets("0x65", mojom::CoinType::SOL, &tokens);
```

Comment:
can we use `mojom::kSolanaMainnet` here? same thing applied to known chain id for the rest of the test

Reply:
https://github.com/brave/brave-core/pull/12905/commits/5af52f331012a256c8d8746c1d0a907e2151974f